src/validation.ts

```diff
@@ -0,0 +1,76 @@
+import { FuncVPCDiscovery, SecurityGroupItem, SubnetItem, VPCDiscovery } from "./types";
```

Comment:
This might be useful to add https://www.serverless.com/framework/docs/providers/aws/guide/plugins#extending-validation-schema

Reply:
I would like to work on in the next release with updates for the `ec2-wrapper.ts`